typescript/bots/ts_libs/utils/native-operators.ts

```diff
@@ -0,0 +1,19 @@
+import { Vector } from "bots/ts_libs/dota";
+
+type InternalScalarType = Vector;
+
+export function sub<T extends InternalScalarType>(a: T, b: T): T {
+    // @ts-ignore
+    return a - b;
+}
```

Comment:
how did you verify if these produce correct values?

Reply:
Yes, it just produces a function that uses that operator, you can see it in diff, also these functions were simply moved from `utils` to here